corehq/util/__init__.py

```diff
@@ -1,5 +1,7 @@
 import re
 
+from .view_utils import reverse
```

Comment:
Yup.
```python
# without internal linking
from corehq.utils.view_utils import reverse
from corehq.utils.dates import unix_time

# if corehq.utils.__init__ imports those two, you can just write
from corehq.utils import reverse, unix_time
```
It lets you present a friendly public API while still having a well organized module.